addons/medical_gui/initSettings.sqf

```diff
@@ -42,3 +42,12 @@
     [0, 10, 3, 1],
     true
 ] call CBA_settings_fnc_init;
+
+[
+    QGVAR(interactionMenuShowTriage),
+    "LIST",
+    [LSTRING(interactionMenuShowTriage_DisplayName), LSTRING(interactionMenuShowTriage_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    [[0, 1, 2], [LELSTRING(common,Disabled), LELSTRING(Medical_Treatment,Anyone), LELSTRING(Medical_Treatment,Medics)], 1],
+    true
```

Comment:
I think this doesn't have to be global (`false` will still allow forcing for those that want that).